application/data/dimensions.py

```diff
@@ -0,0 +1,44 @@
+from application.data.charts import ChartObjectDataBuilder
+from application.data.tables import TableObjectDataBuilder, TableObjectTableBuilder
+
+
+class DimensionObjectBuilder:
+    """
+    Creates an object from table database entries that can be processed using file writers
+    """
+
+    def __init__(self):
+        self.data_table = [[]]
+        self.context = []
+
+    @staticmethod
+    def build(dimension):
+        dimension_object = {"context": DimensionObjectBuilder.get_context(dimension)}
+
+        if dimension.table:
+            dimension_object["table"] = TableObjectDataBuilder.build(dimension.table)
+
+        if dimension.chart:
+            dimension_object["chart"] = ChartObjectDataBuilder.build(dimension.chart)
+
+        if dimension.table:
+            dimension_object["tabular"] = TableObjectTableBuilder.build(dimension.table)
```

Comment:
again I wasn't intending to broaden scope to cover all the classes in data_utils.py but for reference both entries relate to CSV generation
dimension_object["table"] refers to the Source data (CSV)
dimension_object["tabular"] refers to the Download table data (CSV) 
that said it could be the only place we surface this data is in data.json. if we want to get tighter on that file I suggest that either we start talking to potential users but provide information indescribably until we do